### PR TITLE
113 Japanese localization original branch

### LINKED ISSUE #113: Japanese Localization
**Is your feature request related to a problem? Please describe.**
From: WWCodeTokyo From:Tutti-san
> Looking for help with a Japanese translation check for https://speakher.jp, would anyone be able to help?

**Describe the solution you'd like**

1. Japanese translation check for https://speakher.jp
2. Send pull-request to Tutti-san

### PR DESCRIPTION
**Pull request by ann-kilzer**
Resolves #113 

Japanese localization improvements by @shokokb

**Proposed Changes**

Combination of changes from 
- #118 (Please review the comments from this PR)
- #199 
- #197 

**Testing Plan**

Visual check of Netlify build

### CODE REVIEW

File: web/src/i18n/locales/ja.json [hunk 1]
```diff
@@ -5,11 +5,11 @@
         "title": "404：ページが見つかりません"
     },
     "about": {
-        "purpose": "登録者の情報は、イベントの企画者が女性スピーカーを簡単に見つけられるようにするため、また、ご登録いただいた方々が繋がって新しいコミュニティーを作るために使われます。ご注意 : Eメールアドレスを除き登録された情報はWebサイト上に公開されます。公開を望まない情報は共有しないでください。",
+        "purpose": "登録者の情報は、イベントの企画者が女性のスピーカーを簡単に見つけられるようにする目的に使用されます。ご登録いただいた方々がつながって新しいコミュニティーを作るために活用されます。ご注意 : Eメールアドレスを除き登録された情報はWebサイト上に公開されます。公開を望まない情報は共有しないでください。",
```
Comment (ann-kilzer) [inline web/src/i18n/locales/ja.json:8]: Please review the comments: https://github.com/WWCodeTokyo/speak-her-db/pull/118#discussion_r485634258

File: web/src/i18n/locales/ja.json [hunk 2]
```diff
@@ -5,11 +5,11 @@
...
         "summary": {
-            "body": "SpeakHerは日本にいる女性スピーカーを探すデータベースです。\n\n講演会やイベントの企画者は、女性スピーカーの候補者選びに頭を悩ませていると言われています。そのため、イベントやパネルディスカッションなどでスピーチしたい女性スピーカーのデータベースを作ることにしました。こちらのサイトに登録することによって、女性が公の場で広く、多くの人々に発信していく機会を一緒に増やしていきましょう。",
+            "body": "SpeakHerは日本にいる女性のスピーカーを探すデータベースです。\n\n講演会やイベントの企画者は、女性のスピーカーの候補者選びに頭を悩ませていると言われています。そのため、イベントやパネル・ディスカッションなどで講演やスピーチしたい女性のスピーカーのデータベースを作ることにしました。こちらのサイトに登録することにより、女性が公の場で広く、多くの人々に発信していく機会を一緒に増やしていきましょう。",
```
Comment (ann-kilzer) [inline web/src/i18n/locales/ja.json:10]: Please review the comments: https://github.com/WWCodeTokyo/speak-her-db/pull/118#discussion_r485636034

File: web/src/i18n/locales/ja.json [hunk 3]
```diff
@@ -73,7 +73,7 @@
         "email": "Eメール",
         "error": "エラー： {0}",
         "facebook": "FacebookのURL",
-        "jobTitle": "職名",
+        "jobTitle": "役職",
```
Comment (ann-kilzer) [inline web/src/i18n/locales/ja.json:76]: Please review the comments: https://github.com/WWCodeTokyo/speak-her-db/pull/118#discussion_r485637694

File: web/src/i18n/locales/ja.json [hunk 4]
```diff
@@ -87,7 +87,7 @@
         "title": "スピーカーを指名する",
         "topics": {
             "label": "関心のある分野",
-            "hint": "何かを入力してEnterキーを押し、新しいトピックを追加します"
+            "hint": "分野を入力後、Enterキーを押し、新しい分野を追加します"
```
Comment (ann-kilzer) [inline web/src/i18n/locales/ja.json:90]: Please review the comments: https://github.com/WWCodeTokyo/speak-her-db/pull/118#discussion_r485639025

File: web/src/i18n/locales/ja.json (hunk 1 above)
Comment (shokokb) [reply web/src/i18n/locales/ja.json:8]: OK

File: web/src/i18n/locales/ja.json (hunk 2 above)
Comment (shokokb) [reply web/src/i18n/locales/ja.json:10]: OK
データベース database is not familiar with general people.

File: web/src/i18n/locales/ja.json (hunk 3 above)
Comment (shokokb) [reply web/src/i18n/locales/ja.json:76]: OK
プレゼン：presentation
講演：speech

File: web/src/i18n/locales/ja.json (hunk 4 above)
Comment (shokokb) [reply web/src/i18n/locales/ja.json:90]: OK

**Review by shokokb — APPROVED** [review]

File: web/src/i18n/locales/ja.json (hunk 1 above)
Comment (Ania-chan) [reply web/src/i18n/locales/ja.json:8]: @shokokb I'm putting the comment here for your convenience. Please tell me what you think!:
I would add `または、` like `または、ご登録いただいた方々がつながって` before the second sentence. What do you think? I feel it's creating a better connection between the sentences. Also if you think it's better we can remove `Web` from `Webサイト` in this sentence too.

File: web/src/i18n/locales/ja.json [hunk 5]
```diff
@@ -5,11 +5,11 @@
...
     },
     "about": {
-        "purpose": "登録者の情報は、イベントの企画者が女性スピーカーを簡単に見つけられるようにするため、また、ご登録いただいた方々が繋がって新しいコミュニティーを作るために使われます。ご注意 : Eメールアドレスを除き登録された情報はWebサイト上に公開されます。公開を望まない情報は共有しないでください。",
+        "purpose": "登録者の情報は、イベントの企画者が女性のスピーカーを簡単に見つけられるようにする目的に使用されます。ご登録いただいた方々がつながって新しいコミュニティーを作るために活用されます。ご注意 : Eメールアドレスを除き登録された情報はWebサイト上に公開されます。公開を望まない情報は共有しないでください。",
         "summary": {
-            "body": "SpeakHerは日本にいる女性スピーカーを探すデータベースです。\n\n講演会やイベントの企画者は、女性スピーカーの候補者選びに頭を悩ませていると言われています。そのため、イベントやパネルディスカッションなどでスピーチしたい女性スピーカーのデータベースを作ることにしました。こちらのサイトに登録することによって、女性が公の場で広く、多くの人々に発信していく機会を一緒に増やしていきましょう。",
+            "body": "SpeakHerは日本にいる女性のスピーカーを探すデータベースです。\n\n講演会やイベントの企画者は、女性のスピーカーの候補者選びに頭を悩ませていると言われています。そのため、イベントやパネル・ディスカッションなどで講演やスピーチしたい女性のスピーカーのデータベースを作ることにしました。こちらのサイトに登録することにより、女性が公の場で広く、多くの人々に発信していく機会を一緒に増やしていきましょう。",
             "learnMore": "もっと詳しくはこちら",
-            "title": "イベントでプレゼンする女性スピーカーを見つけたいですか？"
+            "title": "イベントで講演していただける女性のスピーカーを見つけたいですか？"
```
Comment (Ania-chan) [inline web/src/i18n/locales/ja.json:12]: @shokokb I was wondering why is it necessary to change `女性のスピーカー` to `女性スピーカー`. I feel the second one sounds more naturally 🤔

File: web/src/i18n/locales/ja.json [hunk 6]
```diff
@@ -35,16 +35,16 @@
     "faq": {
         "correctionA": "こちらまでご連絡ください。{0}",
         "correctionQ": "リストに掲載されている情報の修正・削除を希望される方",
-        "freeA": "このWebサイトを通してスピーカーに連絡することは無料です。しかし、スピーカによって有料で講演を行うスピーカーもいれば、無料で講演を行うスピーカーもいます。私たちは、スピーカーの考えを尊重し、有料であれば時間と労力に合う価格設定をサポートして行きたいと思っています。",
-        "freeQ": "スピーチは無料で行われるのでしょうか？",
-        "obligationA": "私たちのプラットフォームに登録されている女性スピーカーは、必ずしも講演したり、メッセージに返事をしなければならないと言う訳ではありません。私たちのサービスの目的は、イベント主催者と女性スピーカーを繋げることです。そこから、お互いの目的や条件が合うかどうかメッセージをしてご判断ください。",
+        "freeA": "このサイトを介してスピーカーに連絡することは無料です。しかし、スピーカーにより有償の場合、無償の場合があります。私たちは、スピーカーの考えを尊重し、有料であれば時間と労力に合う価格設定をサポートすることを検討しています。",
```
Comment (Ania-chan) [inline web/src/i18n/locales/ja.json:38]: @shokokb Actually this is changing the original meaning of the sentence. `私たちは、スピーカーの考えを尊重し、有料であれば時間と労力に合う価格設定をサポートすることを検討しています。",` It shouldn't mean that `私たちはサポートすることを検討しています。` but  rather `サポートして行きたいと思っています。`

File: web/src/i18n/locales/ja.json [hunk 7]
```diff
@@ -35,16 +35,16 @@
...
+        "freeQ": "講演は無料で行われるのでしょうか？",
+        "obligationA": "私たちのサービスに登録されている女性のスピーカーは、必ずしも講演したり、メッセージに返事をしなければならないというわけではありません。私たちのサービスの目的は、イベント主催者と女性のスピーカーをつなぐことです。そこから、お互いの目的や条件が合うかどうかメッセージをしてご検討ください。",
         "obligationQ": "スピーカーは必ず講演する必要がありますか？",
         "qualifyA": "人前で話す際の最大のハードルは、恐れてしまうことです。パブリック・スピーキングの経験がなくても、人前で話す機会を持ちたいと思っている場合、自分自身をスピーカーとして推薦しても大丈夫です。",
         "qualifyQ": "スピーカーになるための必要なことはありますか？",
         "title": "FAQ",
-        "whoA": "こちらのwebサイトは、Yan Fan、Tutti Quintella、Ann Kilzerらによって作成されたオープンソースプロジェクトです。オリジナルのアイデアは竹村えみミラーさんからです。Woman Who Code Tokyoからも協力を頂きました。こちらのプロジェクトに貢献したい場合は、GitHubページをチェックしてください。",
+        "whoA": "こちらのサイトは、Yan Fan、Tutti Quintella、Ann Kilzerらによって作成されたオープンソースのプロジェクトです。オリジナルのアイデアは竹村えみミラーさんからです。Woman Who Code Tokyoからも協力をいただきました。こちらのプロジェクトに貢献したい場合は、GitHubページをチェックしてください。",
```
Comment (Ania-chan) [inline web/src/i18n/locales/ja.json:45]: Can we change it to `オープンソース・プロジェクト`? what do you think? 🤔

File: web/src/i18n/locales/ja.json (hunk 3 above)
Comment (Ania-chan) [reply web/src/i18n/locales/ja.json:76]: @shokokb Here's the comment:
Is 役職 also appropriate for normal 会社員 jibs that don't have any special title? I changed it to 職名 before to make it more broad.

File: web/src/i18n/locales/ja.json (hunk 4 above)
Comment (Ania-chan) [reply web/src/i18n/locales/ja.json:90]: @shokokb adding my previous comment for your convenience: 
if we change it to 分野 maybe we can write it like 分野を選択後 ? not to make impression that you need to write the whole sentence- you can just select them from the dropdown

**Review by Ania-chan — COMMENTED** [review]
@shokokb I left you a few comments and added the previous ones too since there wasn't any changes after that. Let me know if you have any questions!

File: web/src/i18n/locales/ja.json (hunk 1 above)
Comment (ann-kilzer) [reply web/src/i18n/locales/ja.json:8]: I already linked the prior comment (but didn't copy paste), and it appears Shoko has okayed it. Can you please use [suggested changes](https://thenextweb.com/dd/2018/10/16/github-launches-suggested-changes-to-make-it-easier-to-collaborate-within-pull-requests/) so I know how to appropriately update the branch?

File: web/src/i18n/locales/ja.json (hunk 4 above)
Comment (ann-kilzer) [reply web/src/i18n/locales/ja.json:90]: same here

File: web/src/i18n/locales/ja.json (hunk 7 above)
Comment (ann-kilzer) [inline web/src/i18n/locales/ja.json:45]:
```suggestion
        "whoA": "こちらのサイトは、Yan Fan、Tutti Quintella、Ann Kilzerらによって作成されたオープンソース・プロジェクトです。オリジナルのアイデアは竹村えみミラーさんからです。Woman Who Code Tokyoからも協力をいただきました。こちらのプロジェクトに貢献したい場合は、GitHubページをチェックしてください。",
```

File: web/src/i18n/locales/ja.json (hunk 3 above)
Comment (ann-kilzer) [reply web/src/i18n/locales/ja.json:76]:
```suggestion
        "jobTitle": "職名",
```

File: web/src/i18n/locales/ja.json (hunk 1 above)
Comment (ann-kilzer) [reply web/src/i18n/locales/ja.json:8]: Actually let me try it... Does this look okay to everyone?
```suggestion
        "purpose": "登録者の情報は、イベントの企画者が女性のスピーカーを簡単に見つけられるようにする目的に使用されます。または、ご登録いただいた方々がつながって新しいコミュニティーを作るために活用されます。ご注意 : Eメールアドレスを除き登録された情報はWebサイト上に公開されます。公開を望まない情報は共有しないでください。",
```

Comment (Ania-chan) [reply web/src/i18n/locales/ja.json:8]: @ann-kilzer perfect!

File: web/src/i18n/locales/ja.json (hunk 7 above)
Comment (Ania-chan) [reply web/src/i18n/locales/ja.json:45]: @ann-kilzer yes perfect!

File: web/src/i18n/locales/ja.json (hunk 5 above)
Comment (Ania-chan) [reply web/src/i18n/locales/ja.json:12]:
```suggestion
            "title": "イベントで講演していただける女性スピーカーを見つけたいですか？"
```

File: web/src/i18n/locales/ja.json [hunk 8]
```diff
@@ -20,7 +20,7 @@
     },
     "codeOfConduct": {
         "title": "行動規範",
-        "body": "本ウェブサイトは、女性スピーカーのアピールや募集以外の目的でを利用しないでください。私たちはコミュニティーの安全性を非常に重要視しています。こちらの目的以外の勧誘や嫌がらせはご遠慮ください。"
+        "body": "本ウェブサイトは、女性のスピーカーのアピールや募集以外の目的でを利用しないでください。私たちはコミュニティーの安全性を非常に重要視しています。こちらの目的以外の勧誘や嫌がらせはご遠慮ください。"
```
Comment (Ania-chan) [inline web/src/i18n/locales/ja.json:23]:
```suggestion
        "body": "本ウェブサイトは、女性スピーカーのアピールや募集以外の目的でを利用しないでください。私たちはコミュニティーの安全性を非常に重要視しています。こちらの目的以外の勧誘や嫌がらせはご遠慮ください。"
```

File: web/src/i18n/locales/ja.json (hunk 6 above)
Comment (Ania-chan) [reply web/src/i18n/locales/ja.json:38]:
```suggestion
        "freeA": "このサイトを介してスピーカーに連絡することは無料です。しかし、スピーカーにより有償の場合、無償の場合があります。私たちは、スピーカーの考えを尊重し、有料であれば時間と労力に合う価格設定をサポートして行きたいと思っています。",
```

File: web/src/i18n/locales/ja.json [hunk 9]
```diff
@@ -55,7 +55,7 @@
         "netlify": "Powered by Netlify"
     },
     "hero": {
-        "subtitle": "日本にいる女性スピーカーを探すデータベース",
+        "subtitle": "日本にいる女性のスピーカーを探すデータベース",
```
Comment (Ania-chan) [inline web/src/i18n/locales/ja.json:58]:
```suggestion
        "subtitle": "日本にいる女性スピーカーを探すデータベース",
```

File: web/src/i18n/locales/ja.json (hunk 4 above)
Comment (Ania-chan) [reply web/src/i18n/locales/ja.json:90]:
```suggestion
            "hint": "分野を選択後、Enterキーを押し、新しい分野を追加します"
```

File: web/src/i18n/locales/ja.json (hunk 1 above)
Comment (Ania-chan) [reply web/src/i18n/locales/ja.json:8]:
```suggestion
        "purpose": "登録者の情報は、イベントの企画者が女性スピーカーを簡単に見つけられるようにする目的に使用されます。または、ご登録いただいた方々がつながって新しいコミュニティーを作るために活用されます。ご注意 : Eメールアドレスを除き登録された情報はWebサイト上に公開されます。公開を望まない情報は共有しないでください。",
```

File: web/src/i18n/locales/ja.json (hunk 2 above)
Comment (Ania-chan) [inline web/src/i18n/locales/ja.json:10]:
```suggestion
            "body": "SpeakHerは日本にいる女性のスピーカーを探すデータベースです。\n\n講演会やイベントの企画者は、女性スピーカーの候補者選びに頭を悩ませていると言われています。そのため、イベントやパネル・ディスカッションなどで講演やスピーチしたい女性スピーカーのデータベースを作ることにしました。こちらのサイトに登録することにより、女性が公の場で広く、多くの人々に発信していく機会を一緒に増やしていきましょう。",
```

File: web/src/i18n/locales/ja.json [hunk 10]
```diff
@@ -35,16 +35,16 @@
     "faq": {
         "correctionA": "こちらまでご連絡ください。{0}",
         "correctionQ": "リストに掲載されている情報の修正・削除を希望される方",
-        "freeA": "このWebサイトを通してスピーカーに連絡することは無料です。しかし、スピーカによって有料で講演を行うスピーカーもいれば、無料で講演を行うスピーカーもいます。私たちは、スピーカーの考えを尊重し、有料であれば時間と労力に合う価格設定をサポートして行きたいと思っています。",
-        "freeQ": "スピーチは無料で行われるのでしょうか？",
-        "obligationA": "私たちのプラットフォームに登録されている女性スピーカーは、必ずしも講演したり、メッセージに返事をしなければならないと言う訳ではありません。私たちのサービスの目的は、イベント主催者と女性スピーカーを繋げることです。そこから、お互いの目的や条件が合うかどうかメッセージをしてご判断ください。",
+        "freeA": "このサイトを介してスピーカーに連絡することは無料です。しかし、スピーカーにより有償の場合、無償の場合があります。私たちは、スピーカーの考えを尊重し、有料であれば時間と労力に合う価格設定をサポートすることを検討しています。",
+        "freeQ": "講演は無料で行われるのでしょうか？",
+        "obligationA": "私たちのサービスに登録されている女性のスピーカーは、必ずしも講演したり、メッセージに返事をしなければならないというわけではありません。私たちのサービスの目的は、イベント主催者と女性のスピーカーをつなぐことです。そこから、お互いの目的や条件が合うかどうかメッセージをしてご検討ください。",
```
Comment (Ania-chan) [inline web/src/i18n/locales/ja.json:40]:
```suggestion
        "obligationA": "私たちのサービスに登録されている女性スピーカーは、必ずしも講演したり、メッセージに返事をしなければならないというわけではありません。私たちのサービスの目的は、イベント主催者と女性のスピーカーをつなぐことです。そこから、お互いの目的や条件が合うかどうかメッセージをしてご検討ください。",
```

**Review by Ania-chan — COMMENTED** [review]
@ann-kilzer I added code suggestions (never used it before, it's really cool!)
I only changed the things I already put in my comments. Most places I changed `女性のスピーカー` to `女性スピーカー` thats why there's more suggestions than comments

**Review by Ania-chan — COMMENTED** [review]
@ann-kilzer  Sorry, I missed a few things 🙇

File: web/src/i18n/locales/ja.json [hunk 11]
```diff
@@ -87,7 +87,7 @@
         "title": "スピーカーを指名する",
         "topics": {
             "label": "関心のある分野",
-            "hint": "何かを入力してEnterキーを押し、新しいトピックを追加します"
+            "hint": "分野を選択後、Enterキーを押し、新しい分野を追加します"
```
[the remark this answers is not in the document]
Comment (Ania-chan) [reply web/src/i18n/locales/ja.json:90]: @ann-kilzer I changed it to other way to explain that I thought would be easier to understand

File: web/src/i18n/locales/ja.json (hunk 7 above)
Comment (ann-kilzer) [reply web/src/i18n/locales/ja.json]: done

**Pull request merged**